### PR TITLE
Code-Mixing Filter

### PR DESCRIPTION
**Pull request by cascadianblue**

### CODE REVIEW

File: filters/code_mixing/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Code-Mixing Filter
+
+## What type of a filter is this?
+
+This filter selects for code-mixed texts (i.e. texts using the multiple
```
Comment (sebastianGehrmann) [inline filters/code_mixing/README.md:5]: nit: remove the "the". 

Also maybe elaborate on which languages this works with?

Comment (cascadianblue) [reply filters/code_mixing/README.md]: Fixed typo. I also [mentioned the languages](https://github.com/GEM-benchmark/NL-Augmenter/blob/43aeab14f70877a57f3a7c806a7524c1027148a8/filters/code_mixing/README.md#what-are-the-limitations-of-this-filter) in the README. Is that alright?

**Comment by cascadianblue** [comment]
@sebastianGehrmann I believe I have addressed all of your requested changes. Could you please review again?

**Comment by sebastianGehrmann** [comment]
Thanks, looking good!

**Review by sebastianGehrmann — APPROVED** [review]

**Comment by cascadianblue** [comment]
@sebastianGehrmann thanks for the review. Could you please request review from another contributor? E.g. @kaustubhdhole ?

File: filters/code_mixing/filter.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        "hif",
+        "co",
+        "lrc",
+        "vep",
+        "tyv",
+    ]
+
+    def __init__(self, k=2, threshold=0.5):
+        super().__init__()
+        self.nlp = spacy.load("en_core_web_sm")
```
Comment (kaustubhdhole) [inline filters/code_mixing/filter.py:201]: You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).

File: filters/code_mixing/filter.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        "rue",
+        "xal",
+        "pfl",
+        "dty",
+        "hif",
+        "co",
+        "lrc",
+        "vep",
+        "tyv",
+    ]
```
Comment (kaustubhdhole) [inline filters/code_mixing/filter.py:197]: Please add the revelant [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) too.

**Comment by kaustubhdhole** [comment]
Hi @cascadianblue, this looks good overall. Please address the two comments and you are good to go :) It is okay to skip the evaluation for your PR since we don't have any code-mixed dataset.

**Comment by cascadianblue** [comment]
Okay @kaustubhdhole , I think that takes care of your requested changes

**Comment by kaustubhdhole** [comment]
Thanks for the changes @cascadianblue!

**Review by kaustubhdhole — APPROVED** [review]

**Pull request merged**